### PR TITLE
[Drop-In UI] Changes RoadName positioning and customization

### PR DESCRIPTION
**Pull request by tomaszrybakiewicz**
Closes NAVAND-923

### Description
- Increased `InfoPanel` bottom guideline max pos to 50% of the `NavigationView` height and made it customizable via `ViewStyleCustomization.infoPanelGuidelineMaxPosPercent`

### CODE REVIEW

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt [hunk 1]
```diff
@@ -158,11 +163,14 @@ internal class InfoPanelCoordinator(
     private fun updateGuidelinePosition(
         systemBarsInsets: Insets = context.systemBarsInsets.value,
         infoPanelTop: Int = infoPanel.top,
+        maxPosPercent: Float = context.styles.infoPanelGuidelineMaxPosPercent.value
     ) {
         val parentHeight = (infoPanel.parent as ViewGroup).height
-        val maxPos = (parentHeight * GUIDELINE_MAX_POSITION_PERCENT).toInt()
-        val pos = parentHeight - infoPanelTop - systemBarsInsets.bottom
-        guidelineBottom.setGuidelineEnd(pos.coerceIn(0, maxPos))
+        val maxPos = (parentHeight * maxPosPercent).toInt() - systemBarsInsets.bottom
```
Comment (dzinad) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt:169]: You didn't subtract `systemBarsInsets.bottom` for maxPos before. Is this behaviour change intentional?

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt [hunk 2]
```diff
@@ -81,6 +81,14 @@ internal class InfoPanelCoordinator(
         coroutineScope.launch {
             context.systemBarsInsets.collect { updateGuidelinePosition(systemBarsInsets = it) }
         }
+        coroutineScope.launch {
+            context.styles.infoPanelGuidelineMaxPosPercent.collect {
```
Comment (dzinad) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt:85]: Don't you want to test it?

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt (hunk 1 above)
Comment (tomaszrybakiewicz) [reply libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt:169]: yes

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt (hunk 2 above)
Comment (tomaszrybakiewicz) [reply libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelCoordinator.kt:85]: I tried, but unfortunately, Robolectric was throwing exceptions when I tried to layout the view (`parentLayout.layout(0, 0, 200, 300)`) to get concrete measurements in tests. After fighting with this for over 4 hrs I had to throw the towel.

**Pull request merged**